src/SlashCommands.tsx

```diff
@@ -1054,7 +1054,7 @@ export const Commands = [
     }),
     new Command({
         command: 'discardsession',
-        description: _td('Forces the current outbound group session in an encrypted room to be discarded'),
+        description: _td('Forces the current outbound group device in an encrypted room to be discarded'),
```

Comment:
This string is supposed to be group session not group device. Doing Blind sed is not a good idea.
```suggestion
        description: _td('Forces the current outbound group device in an encrypted room to be discarded'),
```